packages/relay/src/lib/eth.ts

```diff
@@ -1630,7 +1632,7 @@ export class EthImpl implements Eth {
     const params = { timestamp: timestampRangeParams };
 
     // get contract results logs using block timestamp range
-    const logs = await this.common.getLogsWithParams(null, params, requestIdPrefix);
+    const logs = await this.getLogsWithParams(null, params, requestIdPrefix);
```

Comment:
Q: why this revert of the changes made in previous PRs?

Reply:
Backed out by using a new branch.